acceptance/tests/windows/PUP-9719_windows_system_first_pa_run.rb

```diff
@@ -0,0 +1,33 @@
+test_name "PUP-9719 Windows First Agent run as SYSTEM sets cache file permissions correctly" do
+  tag 'risk:medium',
+    'audit:medium',
+    'audit:integration'
+
+  confine :to, :platform => 'windows'
+
+  require 'puppet/acceptance/temp_file_utils'
+  extend Puppet::Acceptance::TempFileUtils
+
+  agents.each do |agent|
+    statedir = on(agent, puppet('config print statedir')).stdout.chomp
+    client_datadir = on(agent, puppet('config print client_datadir')).stdout.chomp
+
+    teardown do
+      on agent, "rm -rf #{statedir}/*"
+      on agent, "rm -rf #{client_datadir}/catalog/*"
+    end
+
+    step "Clean the ProgramData cache directory first" do
+      on agent, "rm -rf #{statedir}/*"
+      on agent, "rm -rf #{client_datadir}/catalog/*"
+    end
+
+    step "Run puppet as System Account" do
+      on agent, 'psexec -s cmd /c "C:\\Program Files\\Puppet Labs\\Puppet\\bin\\puppet.bat" agent -t >> C:\\Windows\\Temp\\Puppet-System-Run.log 2>&1'
```

Comment:
This is still not working as expected, the test pass even without the permission changes.